veracruz-server/src/cli.rs

```diff
@@ -47,27 +46,15 @@ fn main() {
     };
     info!("Loaded policy {}", policy.policy_hash().unwrap_or("???"));
 
-    // create Actix runtime
-    let sys = actix_rt::System::new();
-
     // create Veracruz Server instance
-    let veracruz_server = match veracruz_server::server::server(&policy_json) {
-        Ok(veracruz_server) => veracruz_server,
-        Err(err) => {
-            eprintln!("{}", err);
-            process::exit(1);
-        }
-    };
+    veracruz_server::server::server(&policy_json).unwrap();
 
     println!(
         "Veracruz Server running on {}",
         policy.veracruz_server_url()
     );
-    match sys.block_on(veracruz_server) {
-        Ok(_) => {}
-        Err(err) => {
-            eprintln!("{}", err);
-            process::exit(1);
-        }
+
+    loop {
+        std::thread::sleep(std::time::Duration::MAX);
```

Comment:
I also said this in another PR, but I think this one comes first, so here it is again:
I don't think this is necessary or desirable. If the server exits with an error above, it will panic (and execution will stop). If it exits without an error (not sure if that's possible, really), I would still like the program to stop instead of hang. `sleep` feels wrong here.

Reply:
Got it